src/highspy/highs.py

```diff
@@ -6,13 +6,16 @@
 from threading import Thread, local, RLock, Lock
 from typing import Optional, Any, overload, Callable, Sequence, Mapping, Iterable, SupportsIndex, cast, Union
 
-from ._core import (
-    ObjSense,
+from highspy._core import (
```

Comment:
Irrespective of the naming issue with `ObjSense`, relative imports with `.` are typically worse off than being explicit, i.e. `highspy._core` is better than `._core`